InkyPhat/app/src/main/java/com/novoda/inkyphat/ColorConverter.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.inkyphat;
+
+import android.graphics.Color;
+
+class ColorConverter {
+
+    InkyPhat.Palette convertARBG888Color(int color) {
+        int red = Color.red(color);
+        int blue = Color.blue(color);
+        int green = Color.green(color);
+
+        if (red > 127 && blue > 127 && green > 127) {
```

Comment:
yeah, like an advanced feature. It's a trade off of configuration vs simplicity to use.

yeah if I made them a config in this class, then the factory could have another builder that took more options